pdcupdater/handlers/depchain/rpms.py

```diff
@@ -0,0 +1,249 @@
+""" Handlers for an RPM dependency chain model.
+
+The idea here is that these are two of what will eventually be many handlers
+that build and maintain a dependency chain in PDC.
+
+There are two handlers here.  They both respond to koji 'tag' events. One of
+them figures out what the build required at *build* time, and updates PDC with
+that. The other figures out what the build requires at *run* time, and updates
+PDC with that.
+
+"""
+
+import logging
+import time
+
+import pdcupdater.handlers
+import pdcupdater.services
+from pdcupdater.utils import (
+    rawhide_tag,
+    tag2release,
+)
+
+
+log = logging.getLogger(__name__)
+
+
+def interesting_tags():
```

Comment:
Actually, there's another copy of it in another file::

```
pdcupdater/handlers/depchain/rpms.py:def interesting_tags():
pdcupdater/handlers/depchain/rpms.py:        interesting = interesting_tags()
pdcupdater/handlers/depchain/rpms.py:        tags = interesting_tags()
pdcupdater/handlers/depchain/rpms.py:        tags = interesting_tags()
pdcupdater/handlers/rpms.py:def interesting_tags():
pdcupdater/handlers/rpms.py:        interesting = interesting_tags()
pdcupdater/handlers/rpms.py:            for tag in interesting_tags()
```

We should consolidate them and move them to ``pdcupdater.utils`` I think.